clang/test/CodeGenSYCL/debug-info-checksum.cpp

```diff
@@ -0,0 +1,9 @@
+// RUN: %clangxx -fsycl -fsycl-device-only -I %S %S/Inputs/debug-info-checksum.cpp \
```

Comment:
I don't think that using driver in frontend tests is the right thing to do.

Reply:
Moreover, I've just realized that this tests doesn't check what I want it to check: it checks checksum emitted in device code, whilst what I'm fixing is checksum calculation on host

Reply:
Indeed, I've changed the test in a54b24a9bdcac9a977afc73d00540d0657c4255f